router/core/graphql_prehandler.go

```diff
@@ -423,20 +447,33 @@ func (h *PreHandler) handleOperation(req *http.Request, buf *bytes.Buffer, httpO
 		)
 
 		httpOperation.traceTimings.StartParse()
+		startParsing := time.Now()
```

Comment:
I intentionally did not make this conditionally because the performance is negligible and I prefer readability in this case.